### PR TITLE
Adding basic threads code

### PR DESCRIPTION
**Pull request by elenaperezrioja**
Basic first version of using experimental Threads component. There are major things missing here: scrolling down automatically, date pills, new message indicators,... but the barebones are there.

### CODE REVIEW

**Comment by elenaperezrioja** [comment]
* **#173** <a href="https://app.graphite.dev/github/pr/getcord/clack/173?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @elenaperezrioja and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/client/components/Threads.tsx [hunk 1]
```diff
@@ -208,3 +231,59 @@ const XIcon = styled(XMarkIcon)({
...
+        threadsData={threadsDataReversed}
+        composerOptions={composerOptions}
+        replace={REPLACE}
+      />
+    </OpenThreadContext.Provider>
+  );
+}
+
+const StyledExperimentalThreads = styled(experimental.Threads)({
+  '&': { overflowY: 'auto' },
```
Comment (AlbertQM) [inline src/client/components/Threads.tsx:279]: Just a q: why overflowY, instead of `maxHeight: 100%`?

File: src/client/components/Threads.tsx [hunk 2]
```diff
@@ -208,3 +231,59 @@ const XIcon = styled(XMarkIcon)({
...
+  '& .cord-inline-reply-button': { display: 'none' },
+  '&.cord-threads': { border: 'none', padding: 0 },
```
Comment (AlbertQM) [inline src/client/components/Threads.tsx:281]: just a q (I never used styled components): why &.cord-threads? Isn't & already cord-threads?

**Review by AlbertQM — APPROVED** [review]

File: src/client/components/Threads.tsx (hunk 2 above)
Comment (elenaperezrioja) [reply src/client/components/Threads.tsx:281]: The sdk styles override it, but I can just merge both and use && for better specificity

**Pull request merged**